package.json

```diff
@@ -9,7 +9,8 @@
   "scripts": {
     "watch": "webpack --config ./webpack.config.js --watch",
     "build": "webpack --config ./webpack.config.js",
-    "test": "karma start"
+    "test": "karma start",
+    "lint": "eslint normandy client webpack.config.js karam.conf.js"
```

Comment:
it makes the command always pass so that npm doesn't error out but you still get all the lint output. you just don't get all the `npm ERR!` output and it doesn't spit out an `npm-debug.log` file each time you run.